locales/templates/app.pot

```diff
@@ -305,6 +305,140 @@ msgctxt "preference_about"
 msgid "About %1$s"
 msgstr ""
 
+#. Header for the list of installed search engines
+msgctxt "preference_search_installed_search_engines"
+msgid "Installed search engines"
+msgstr ""
+
+#. Action for restoring the default list of search engines
+msgctxt "preference_search_restore"
+msgid "Restore default search engines"
+msgstr ""
+
+#. Action for adding an additional search engine to the list
+msgctxt "preference_search_add"
+msgid "Add another search engine"
+msgstr ""
+
+msgctxt "preference_search_remove"
+msgid "Remove"
+msgstr ""
+
+#. Clickable text in tutorial to open a dialog to manually add a search engine
+msgctxt "tutorial_search_manually_add"
+msgid "MANUALLY ADD"
```

Comment:
Are we sure there's a reason this is all in caplocks? I don't think I've ever seen this this way in any of our Focus strings before, so want to be extra sure

Reply:
Oh! Okay, in the past for Fennec, when we've wanted strings that are in all caps, I think @flodolo has asked for them to be made all caps in the dtd files too - I was following that but I can do whatever really.

Reply:
Ok! Just wanted to be sure because I don't recall doing this before on Focus. But if we're doing this in other mobile projects, than it makes sense to do it this way here too. Thanks for explaining!

Reply:
Side note: I've seen locales completely missing the fact that they're supposed to translate strings in all caps. The good news is that we're going to have Focus in Transvision soon, so that'll be easy to check.

Reply:
Note that Android will capitalize buttons etc. at runtime (depending on theme and for languages where it makes sense). So in the past I did not capitalize the strings itself in Focus.